_docs/tasks/traffic-management/services-expansion.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Accessing Services in the Expanded Mesh
+overview: This task shows you how to use services provided by VM
+
+order: 60
+
+#draft: true
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows you how to configure services running in a VM that joined the cluster.
+
+Current task was tested on GCP. _WIP on adding specific info for other providers_
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](({{home}}/docs/setup/).
+
+* Deploy the [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application.
+
+* Create a VM named 'db' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
+
+## Running mysql on the VM
+
+We will first install mysql on the VM, and configure it as a backend for the ratings service.
+
+On the VM:
+```bash
+  sudo apt-get update && apt-get install ...
+  # TODO copy or link the istio/istio test script
+```
+
+## Registering the mysql service with the mesh
+
+### Machine admin
+First step is to configure the VM sidecar, by adding the service port and restarting the sidecar.
```

Comment:
The first step when registering the mysql service with the mesh is to configure the VM's Envoy sidecar proxy with the appropriate port. You do this by adding the service port and restarting the sidecar.